### PR TITLE
Added createPost command.

### LINKED ISSUE #21: `publishPost`

### PR DESCRIPTION
**Pull request by iamdharmesh**
### Description of the Change
Added command for creating a post.
`createPost` command accepts below post data
- `postType`: To create a post for the specific post type.
- `title`: Post Title
- `content`: Post content
- `status`: Post status (`draft` and `publish` are supported for now.)

_Note: This command is for Block Editor workflow only._

Closes #21

### Checklist:

- [x] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my change.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/commands/create-post.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+ *          `postType` - Post type,
+ *          `title` - Post Title,
+ *          `content` - Post Content,
+ *          `status` - Post Status
+ *        }
+ *
+ * @example
+ * Create a Post
+ * ```
+ * cy.publishPost({
```
Comment (cadic) [inline src/commands/create-post.ts]: Shouldn't it say `cy.createPost` and also for next examples?

File: src/commands/create-post.ts [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  title = 'Test Post',
+  content = 'Test content',
+  status = 'publish',
+}: {
+  title: string;
+  postType?: string;
+  content?: string;
+  status?: string;
+}): void => {
+  cy.log(postType);
```
Comment (cadic) [inline src/commands/create-post.ts]: Is this log output required for the production?

**Review by cadic — CHANGES_REQUESTED** [review]
@iamdharmesh thank you for your great work on this. I just have 2 small comments on documenting mistype and logging output

File: src/commands/create-post.ts (hunk 2 above)
Comment (iamdharmesh) [reply src/commands/create-post.ts]: Removed :+1:

File: src/commands/create-post.ts (hunk 1 above)
Comment (iamdharmesh) [reply src/commands/create-post.ts]: Good Catch. Thanks for this!

**Review by cadic — APPROVED** [review]
LGTM!

**Pull request merged**